projects.json

```diff
@@ -5,6 +5,9 @@
       "name": "Project name goes here",
       "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi Ut aliquip ex ea commodo consequat.  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.",
```

Comment:
- [ ] Please try as much as possible to customize the portifolio. In this case, I kindly suggest to customize the projects with different project names. This will make your portifolio professional with good UI ☺️.

Reply:
thankyou @AlphaNtihinduka for your suggestion, I will consider it, to write different project names as I add real projects of my own instead of mockups provided by Figma.